src/builtin.js

```diff
@@ -798,7 +798,10 @@ Sk.builtin.exec = function (code, globals, locals) {
     globals = globals || tmp;
     return Sk.misceval.chain(
         code,
-        (co) => eval(co.code)(globals, locals),
+        (co) => {
+            co.code += "\n" + co.funcname + ";";
```

Comment:
what does this do? can you add a comment?

Reply:
This ensures that the main function scope within the compiled code string is returned from eval. 

What do you think about moving this line to the end of the Sk.compile instead?

The line also appears in import.js and so we can then only append this line in import.js if we're importing a $builtinmodule. 

That might cleaner?

Reply:
yeah, makes sense to me keep all the code producey bits in there